solidity/ecdsa/contracts/libraries/Wallets.sol

```diff
@@ -81,11 +81,24 @@ library Wallets {
         return self.registry[walletID].publicKeyX != bytes32(0);
     }
 
+    /// @notice Returns Keccak256 hash of the wallet signing group members
+    ///         identifiers array. Group members do not include operators
+    ///         selected by the sortition pool that misbehaved during DKG.
+    /// @param walletID ID of the wallet
+    /// @return Wallet signing group members hash
+    function getWalletMembersIdsHash(Data storage self, bytes32 walletID)
```

Comment:
Worth documenting that this method will return `bytes32(0)` if `walletID` refers to an unknown wallet.

Reply:
Done in #2908, commit https://github.com/keep-network/keep-core/pull/2908/commits/b9898f4d6eace62316d9bb512a94559d7f4f96a0.